### PR TITLE
[#1414] Word Cloud Word Frequency Selection

### PR DESCRIPTION
**Pull request by mohamagdy**
## What type of PR is this?

- [x] Feature

## Description

This PR addresses part of https://github.com/getredash/redash/issues/1414 which is the ability to pass the frequency when creating "Word Cloud"

## Related Tickets & Documents

Related GitHub issue https://github.com/getredash/redash/issues/1414

### CODE REVIEW

**Comment by truebit** [comment]
Tried applying the patch on Redash 7.0.0, but the world cloud visualizations does not change. Do you know why? 
I applied the patch, then restarted server and celery, but got no luck.

**Comment by mohamagdy** [comment]
@truebit I can see there are some merge conflicts with the files changed in this pull request. @kravets-levko might have introduced a new algorithm for showing words in the Word Cloud in the changes he applied but not sure.

If you have not applied @kravets-levko changes to the files `client/app/visualizations/word-cloud/index.js` and `client/app/visualizations/word-cloud/word-cloud-editor.html` then I can help with this.

**Comment by truebit** [comment]
I figured out, need to `npm run build` in `client` directory to apply the changes.
Thanks @mohamagdy

File: client/app/visualizations/word-cloud/word-cloud-editor.html [hunk 1]
```diff
@@ -5,4 +5,10 @@
       <select ng-options="name for name in queryResult.getColumnNames()" ng-model="visualization.options.column" class="form-control"></select>
     </div>
   </div>
+  <div class="form-group">
+    <label class="col-lg-6">Word Cloud Frequency Column Name</label>
+    <div class="col-lg-6">
+      <select ng-options="name for name in [''].concat(queryResult.getColumnNames())" ng-model="visualization.options.frequency" class="form-control"></select>
```
Comment (truebit) [inline client/app/visualizations/word-cloud/word-cloud-editor.html]: There will be an extra blank option when creating the world cloud like below:
图片

after saving it, this symptom will disappear.

**Comment by arikfr** [comment]
Thank you for your effort on this!

As this was implemented as part of the React migration in #3930 closing this PR.

**Pull request closed without merging**